frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf

```diff
@@ -106,7 +130,7 @@
             });
             </core_rt:forEach>
 
-            projectVulnerabilityTable = $('#vulnerabilityTable').DataTable({
+            projectVulnerabilityTable = $('#vulnerabilityTable').dataTable({
```

Comment:
Are you sure you didn't break line 155 by that? Because `DataTable` and `dataTable` return different objects, AFAIK.

Reply:
This file is present multiple times in our project (which the columns slightly difference, but the bulk is the same). In the other `vulnerabilities.jspf`, `dataTable` is used instead of `DataTable` and the tooltips are working. Using `DataTable` was creating other problems for me.

Reply:
If the tooltips work, then all is fine